autonomy/cli/scaffold_fsm.py

```diff
@@ -0,0 +1,589 @@
+    AbstractRound,
+    DegenerateRound,
+)
+
+
+FILE_HEADER = """
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#   Copyright 2018-2021 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""
+
+ROUNDS_FILENAME = "rounds.py"
+BEHAVIOURS_FILENAME = "behaviours.py"
+MODELS_FILENAME = "models.py"
+HANDLERS_FILENAME = "handlers.py"
+
+
+def _remove_quotes(input_str: str) -> str:
+    """Remove single or double quotes from a string."""
+    return input_str.replace("'", "").replace('"', "")
+
+
+def _get_abci_app_cls_name_from_dfa(dfa: DFA) -> str:
+    """Get the Abci app class name from a DFA object."""
+    return dfa.label.split(".")[-1]
+
+
+def _try_get_behaviour_cls_name_from_round_cls_name(round_cls_name: str) -> str:
+    """
+    Try to get the behaviour class name from the round class name.
+
+    It consits in trying to replace the suffix "Round" with "Behaviour".
+
+    :param round_cls_name: the round class name
+    :return: the new behaviour class name
+    """
+    return re.sub("(.*)Round", "\\1Behaviour", round_cls_name)
+
+
+def _try_get_round_behaviour_cls_name_from_abci_app_cls_name(
+    abci_app_cls_name: str,
+) -> str:
+    """
+    Try to get the round behaviour class name from the Abci app class name.
+
+    It consits in trying to replace the suffix "AbciApp" with "RoundBehaviour".
+
+    :param abci_app_cls_name: the abci app class name
+    :return: the new round behaviour class name
+    """
+    return re.sub("(.*)AbciApp", "\\1RoundBehaviour", abci_app_cls_name)
+
+
+class AbstractFileGenerator(ABC):
+    """An abstract class for file generators."""
+
+    FILENAME: str
+
+    def __init__(self, dfa: DFA) -> None:
+        """Initialize the abstract file generator."""
+        self.dfa = dfa
+
+    @abstractmethod
+    def get_file(self) -> str:
+        """Get file content."""
+
+    def write_file(self, output_dir: Path) -> None:
+        """Write the file to output_dir/FILENAME."""
+        (output_dir / self.FILENAME).write_text(self.get_file())
+
+
+class RoundFileGenerator(AbstractFileGenerator):
+    """File generator for 'rounds.py' modules."""
+
+    FILENAME = ROUNDS_FILENAME
+
+    ROUNDS_FILE_HEADER = dedent(
+        """\
+        \"\"\"This package contains a scaffold of rounds.\"\"\"
```

Comment:
```suggestion
        \"\"\"This package contains the rounds of {FSM_NAME}.\"\"\"
```
People usually don't update these, and so then it says scaffold everywhere :D